### ISSUE TITLE
support for brew/cask versions?

### ISSUE THREAD

**Issue by AnalogJ** [AnalogJ/capsulecd#9]

**Comment by AnalogJ** [comment]
not sure this makes sense as brew/cask versions are commited to shared repo. CapsuleCD could handle RPM/DEB files though.. would it make sense to fight with Packer though?

**Comment by bby-bishopclark** [comment]
FPM discards the source artefact, and that's a concern in many places.  Sure, the source is there in the code, but having the source as-is and requiring an extra tool to generate the source artefact is enough to exclude it from distribution into some organizations.

Generating the source and binary RPM distributable artefacts via 'rpm-build -tb tarball.gz' and 'rpm-build -ts tarball.gz' is eeeasy to do as a build step, as long as we can use calsuleCD to get us that project-version.tgz with the right project-ver/files layout; and having the source compatible with such a pipeline retains that 'self-hosted' or 'self-contained' methodology to achieve adequacy among those who wouldn't accept it without.

TL;DR - FPM provides a small benefit, and at great hidden cost.
